src/NewWindow.js

```diff
@@ -153,12 +165,28 @@ function copyStyles(source, target) {
   Array.from(source.styleSheets).forEach(styleSheet => {
 
     // For <style> elements
-    if (styleSheet.cssRules) {
+    let rules;
+    try {
+      rules = styleSheet.cssRules;
+    } catch (err) {
+      console.error(err);
+    }
+    if (rules) {
       const newStyleEl = source.createElement('style')
 
       // Write the text of each rule into the body of the style element
       Array.from(styleSheet.cssRules).forEach(cssRule => {
-        newStyleEl.appendChild(source.createTextNode(cssRule.cssText))
+        const { cssText, type } = cssRule;
+        let returnText = cssText;
+        if ([3, 5].includes(type)) {
```

Comment:
@Moshem123 can we make it clear what does `3` and `5` means? Probably, declaring them as constant and giving a self explaining name. 🙇